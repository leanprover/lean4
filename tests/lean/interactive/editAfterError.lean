#check tru
#check fal
        --^ insert: "s"
        --^ collectDiagnostics
