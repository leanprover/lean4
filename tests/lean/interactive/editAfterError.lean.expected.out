{"textDocument": {"version": 2, "uri": "file://editAfterError.lean"},
 "contentChanges":
 [{"text": "s",
   "range":
   {"start": {"line": 1, "character": 10},
    "end": {"line": 1, "character": 11}}}]}
{"version": 2,
 "uri": "file://editAfterError.lean",
 "diagnostics":
 [{"source": "Lean 4",
   "severity": 1,
   "range":
   {"start": {"line": 0, "character": 7}, "end": {"line": 0, "character": 10}},
   "message": "unknown identifier 'tru'",
   "fullRange":
   {"start": {"line": 0, "character": 7}, "end": {"line": 0, "character": 10}}},
  {"source": "Lean 4",
   "severity": 1,
   "range":
   {"start": {"line": 1, "character": 7}, "end": {"line": 1, "character": 11}},
   "message": "unknown identifier 'fals'",
   "fullRange":
   {"start": {"line": 1, "character": 7},
    "end": {"line": 1, "character": 11}}}]}
{"version": 2,
 "uri": "file://editAfterError.lean",
 "diagnostics":
 [{"source": "Lean 4",
   "severity": 1,
   "range":
   {"start": {"line": 0, "character": 7}, "end": {"line": 0, "character": 10}},
   "message": "unknown identifier 'tru'",
   "fullRange":
   {"start": {"line": 0, "character": 7}, "end": {"line": 0, "character": 10}}},
  {"source": "Lean 4",
   "severity": 1,
   "range":
   {"start": {"line": 1, "character": 7}, "end": {"line": 1, "character": 11}},
   "message": "unknown identifier 'fals'",
   "fullRange":
   {"start": {"line": 1, "character": 7},
    "end": {"line": 1, "character": 11}}}]}
{"version": 2,
 "uri": "file://editAfterError.lean",
 "diagnostics":
 [{"source": "Lean 4",
   "severity": 1,
   "range":
   {"start": {"line": 0, "character": 7}, "end": {"line": 0, "character": 10}},
   "message": "unknown identifier 'tru'",
   "fullRange":
   {"start": {"line": 0, "character": 7}, "end": {"line": 0, "character": 10}}},
  {"source": "Lean 4",
   "severity": 1,
   "range":
   {"start": {"line": 1, "character": 7}, "end": {"line": 1, "character": 11}},
   "message": "unknown identifier 'fals'",
   "fullRange":
   {"start": {"line": 1, "character": 7},
    "end": {"line": 1, "character": 11}}}]}
